### PR TITLE
budget insulateds should no longer be immune to shocks

### PR DESCRIPTION
**Pull request by SomeguyManperson**
I have reduced the amount of coding errors by reducing the amount of code
:cl:  
bugfix: budget insulateds now die on being shocked a couple times
/:cl:

### CODE REVIEW

File: code/modules/clothing/gloves/color.dm [hunk 1]
```diff
@@ -23,10 +23,10 @@
 	if(damaged)
 		qdel(src)
 		new/obj/effect/decal/cleanable/ash(src)
```
Comment (redmoogle) [inline code/modules/clothing/gloves/color.dm]:
```suggestion
		new/obj/effect/decal/cleanable/ash(src)
		qdel(src)
```

File: code/modules/clothing/gloves/color.dm [hunk 2]
```diff
@@ -23,10 +23,10 @@
...
-		to_chat(src, "<span class='notice'>Your gloves catch fire and disintegrate!</span>")
+		visible_message("<span class='notice'>Your gloves catch fire and disintegrate!</span>")
 	else
 		damaged = TRUE
-		to_chat(src, "<span class='notice'>Your gloves begin to melt!</span>")
+		visible_message("<span class='notice'>Your gloves begin to melt!</span>")
```
Comment (redmoogle) [inline code/modules/clothing/gloves/color.dm]: Doesnt make any sense, other people see this

**Review by redmoogle — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 3]
```diff
@@ -484,8 +484,8 @@
 			var/obj/item/clothing/gloves/G = gloves
 			if(G.siemens_coefficient <= 0)
 				total_coeff -= 0.5
-			var/obj/item/clothing/gloves/color/fyellow/greytide = G
-			if(istype(greytide))
+			if(istype(G, /obj/item/clothing/gloves/color/fyellow)
```
Comment (redmoogle) [inline code/modules/mob/living/carbon/human/human_defense.dm]: also this
```suggestion
			if(istype(G, /obj/item/clothing/gloves/color/fyellow))
```

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 4]
```diff
@@ -501,8 +501,8 @@
 		if(gloves)
 			var/obj/item/clothing/gloves/G = gloves
 			gloves_siemens_coeff = G.siemens_coefficient
-			var/obj/item/clothing/gloves/color/fyellow/greytide = G
-			if(istype(greytide))
+			if(istype(G, /obj/item/clothing/gloves/color/fyellow)
```
Comment (redmoogle) [inline code/modules/mob/living/carbon/human/human_defense.dm]: and this
```suggestion
			if(istype(G, /obj/item/clothing/gloves/color/fyellow))
```

**Review by redmoogle — CHANGES_REQUESTED** [review]

**Comment by ToasterBiome** [comment]
UnU now im gonna have to harass cargo or engineering

**Review by redmoogle — APPROVED** [review]

**Comment by N3D6** [comment]
pog

**Pull request merged**